diff --git a/node_modules/youtubei.js/dist/src/core/Player.js b/node_modules/youtubei.js/dist/src/core/Player.js
index 4dabe22..b252229 100644
--- a/node_modules/youtubei.js/dist/src/core/Player.js
+++ b/node_modules/youtubei.js/dist/src/core/Player.js
@@ -12,13 +12,17 @@ export default class Player {
         this.nsig_sc = nsig_sc;
         this.sig_sc = sig_sc;
     }
-    static async create(cache, fetch = Platform.shim.fetch, po_token) {
+
+    static async create2(cache, playerID) {
+        return this.create(cache, Platform.shim.fetch, undefined, playerID);
+    }
+    static async create(cache, fetch = Platform.shim.fetch, po_token, playerID) {
         const url = new URL('/iframe_api', Constants.URLS.YT_BASE);
         const res = await fetch(url);
         if (!res.ok)
             throw new PlayerError(`Failed to get player id: ${res.status} (${res.statusText})`);
         const js = await res.text();
-        const player_id = getStringBetweenStrings(js, 'player\\/', '\\/');
+        const player_id = playerID ?? getStringBetweenStrings(js, 'player\\/', '\\/');
         Log.info(TAG, `Got player id (${player_id}). Checking for cached players..`);
         if (!player_id)
             throw new PlayerError('Failed to get player id');
